include/secp256k1.h

```diff
@@ -147,6 +147,15 @@ typedef int (*secp256k1_nonce_function)(
      * 1. If using Libtool, it defines DLL_EXPORT automatically.
      * 2. In other cases, SECP256K1_DLL_EXPORT must be defined. */
 #   define SECP256K1_API extern __declspec (dllexport)
+#  else
+    /* Building libsecp256k1 as a static library on Windows.
```

Comment:
nit: Referencing the use of a static library does not apply to our cases with the `tests` and `noverify_tests` targets, where object files are linked directly.

Reply:
You mean https://github.com/bitcoin-core/secp256k1/blob/master/src/CMakeLists.txt#L89, right? True, and it's also true for `bench_internal` and `bench_ecmult`. 

Would you prefer "Building libsecp256k1 not as a DLL."? (I dropped the Windows part because it's anyway only Windows in the outer #ifdef.)

Reply:
> Looks good. However, I'm not sure if it worth invalidating all ACKs.

Ok, let's just get it merged.